app/javascript/application.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (<h1>Hello Word!</h1>);
+}
+
+ReactDOM.render(
+  <App/>,
+  document.getElementById('root'),
+);
```

Comment:
- [ ] Great job working on the backend side but I think it would be great if you work on the front end side to display the messages since it is mentioned in the requirements to make a component and redux reducer and action and store so kindly add these requirements to display the messages 🔥✨💪🏻♻

- Instead of using our views to display the messages we should use react component ♻🚀🔥
![image](https://github.com/paulshaun-kdtk/hello-react-in-rails/assets/58553711/c8015950-6478-41bd-953f-20ad9173875a)